go.mod

```diff
@@ -1,22 +1,64 @@
 module github.com/firehydrant/terraform-provider-firehydrant
 
-go 1.16
+go 1.18
```

Comment:
@splittingred is it necessary to bump the go version for this PR?
Would it be possible to open a separate PR with just the version change?

Thanks!

Reply:
https://github.com/firehydrant/terraform-provider-firehydrant/pull/125